backoffice/src/views/SingleBeaconRecordView.tsx

```diff
@@ -161,7 +161,7 @@ export const SingleBeaconRecordView: FunctionComponent<
         </OnlyVisibleToUsersWith>
         <OnlyVisibleToUsersWith role={"DELETE_BEACONS"}>
           {beacon.status !== BeaconStatuses.Deleted && (
-            <span className={classes.button}>
+            <span className="hidden-feature">
```

Comment:
Probably want to keep both of these when switching back?

            <span className={classes.button + "hidden-feature"} >

Reply:
Or just comment it out ?